extensions/amp-analytics/0.1/variables.js

```diff
@@ -351,3 +355,18 @@ export function getConsentStateStr(element) {
     return EXTERNAL_CONSENT_POLICY_STATE_STRING[consent];
   });
 }
+
+/**
+ * Converts string to boolean
+ * @param {string} str
+ * @return {boolean}
+ */
+function stringToBool_(str) {
+  return (
+    str !== 'false' &&
```

Comment:
nit: No need to add the trailing `_` to local function. And can we please also use this helper method here? Thanks 
https://github.com/ampproject/amphtml/blob/2250345ce3e6d2ab903538561f0bd6124b542001/extensions/amp-analytics/0.1/amp-analytics.js#L727-L734